src/_pytest/config/__init__.py

```diff
@@ -267,10 +267,13 @@ def parse_hookimpl_opts(self, plugin, name):
         # collect unmarked hooks as long as they have the `pytest_' prefix
         if opts is None and name.startswith("pytest_"):
             opts = {}
-
         if opts is not None:
+            # TODO: DeprecationWarning, people should use hookimpl
```

Comment:
I suggest we deprecate this later down the road instead of in 4.1, so let's create an issue so we don't lose track of this. Also add a reference to the issue # in this comment.

Reply:
Oh the issue is there already: #4562